src/header.rs

```diff
@@ -27,6 +27,16 @@ pub enum Bpp {
     Bits32,
 }
 
+/// Image row order
+#[derive(Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Debug)]
+#[non_exhaustive]
+pub enum RowOrder {
+    /// Bottom-up (standard)
+    BottomUp,
```

Comment:
If there is a defined default value for an enum it's a good idea to implement the `Default` trait:
```rust
impl Default for RowOrder {
    fn default() -> Self {
        Self::BottomUp
    }
}
```